cli/talekclient/main.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/privacylab/talek/common"
+	"github.com/privacylab/talek/libtalek"
+)
+
+var config = flag.String("config", "talek.conf", "Client configuration for talek")
```

Comment:
e.g. when I run this command without arguments, I get "panic: runtime error: invalid memory address or nil pointer dereference, /home/ryscheng/go/src/github.com/privacylab/talek/cli/talekclient/main.go:29 +0x60"